ui/js/rewards.js

```diff
@@ -7,13 +7,13 @@ import {
 
 function rewardMessage(type, amount) {
   return {
-    new_developer: `You earned ${amount} for registering as a new developer.`,
-    new_user: `You earned ${amount} LBC new user reward.`,
-    confirm_email: `You earned ${amount} LBC for verifying your email address.`,
-    new_channel: `You earned ${amount} LBC for creating a publisher identity.`,
-    first_stream: `You earned ${amount} LBC for streaming your first video.`,
-    many_downloads: `You earned ${amount} LBC for downloading some of the things.`,
-    first_publish: `You earned ${amount} LBC for making your first publication.`,
+    new_developer: __(`You earned %s for registering as a new developer.`, amount),
```

Comment:
Tick marks are only used around interpolated strings. If interpolation is being dropped, please switch to double quotes.